src/main/java/com/thullo/web/payload/request/UpdateBoardRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.thullo.web.payload.request;
```

Comment:
I don't think there is a need for this class, `BoardRequest` can the job that this class does